<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DDMainMenuWindow</class>
 <widget class="QMainWindow" name="DDMainMenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>NAME: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>5</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>CLASS:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="classSelectionBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rogue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wizard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cleric</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fighter</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>50</y>
      <width>850</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="statsTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>300</width>
      <height>205</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <row>
     <property name="text">
      <string>STRENGTH</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DEXTERITY</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>CONSTITUTION</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>INTELLIGENCE</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>WISDOM</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>CHARISMA</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stat Bonus</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="statsRollButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll the stats</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="savingThrowTable">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>180</width>
      <height>205</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Strength</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Dexterity</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Constitution</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Intelligence</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Wisdom</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Charisma</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Save Bonus</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>300</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Character Attributes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Saving Throws</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>850</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Hit Points</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hitPointsLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>5</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>RACE:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="raceSelectionBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>== Select Character Race ==</string>
    </property>
    <item>
     <property name="text">
      <string>Human</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Elf</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dwarf</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Halfling</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dragonborn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thieflieng</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Half-Orc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Half-Elf</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>850</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="genCharSheet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Character Sheet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hpReRoll">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Re-Roll HP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="characterNameEdit">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
